cmd/version.go

```diff
@@ -8,8 +8,9 @@ import (
 )
 
 var (
-	Version string
-	commit  = "none"
+	Version   string
+	GitCommit string
+	commit    = "none"
```

Comment:
We can set GitCommit by default I think.

Something similar to `var GitCommit string = "none"`
@ArmanMazdaee better syntax or general suggestions?

Reply:
Also removed the `commit` var